src/components/auth/Login.js

```diff
@@ -1,8 +1,60 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
+import { withRouter, Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { loginUser } from '../../actions/authActions';
+
+export class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      emailError:'',
+      passwordError:'',
+      notFoundUser:''
+    };
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+  componentWillReceiveProps(nextProps) {
+   
+    if (nextProps.errors) {
+      if(nextProps.errors.email){
+        this.setState({emailError:nextProps.errors.email})
+      }
+      else if(nextProps.errors.password){
+        this.setState({passwordError:nextProps.errors.password})
+      }
+      else{
+        this.setState({notFoundUser:nextProps.errors.error})
+      }
+      console.log('NEXT PROPS',nextProps.errors)
+      // this.setState({ errors: nextProps.errors.errors });
```

Comment:
Please remove commented out code

Reply:
And the console.log too

Reply:
I have worked on that. Thank you for the feedback